guides/V1_space_template_info.md

```diff
@@ -0,0 +1,93 @@
+---
+copyright: 'Copyright IBM Corp. 2018'
+link: 'space-template-info'
+is: 'experimental'
+---
+
+# Space Template Info
+
+In addition to those space properties documented publicly, you can add _templateInfo_ to your _space_ request in order to fetch
+information about the template used to create a space.
```

Comment:
> those space properties documented publicly

It's not clear what this refers to.

> information about the template used to create a space.

How about 

> [...] the template associated with the space

Reply:
Amended. (Was referring to properties in the "published" Space guide, but removed that piece.)